index.html

```diff
@@ -35,6 +35,20 @@
             des <a href="http://codefor.de/berlin">Open Knowledge Lab Berlin</a>.
             <span id="dataSource">Datenquelle</span>. <a href="https://github.com/wo-ist-markt/wo-ist-markt.github.io">Code</a> auf GitHub.</p>
         </div>
+        <!-- Piwik -->
+        <script type="text/javascript">
+          var _paq = _paq || [];
+          _paq.push(['trackPageView']);
+          _paq.push(['enableLinkTracking']);
+          (function() {
+            var u="//piwik.okfn.de/";
+            _paq.push(['setTrackerUrl', u+'piwik.php']);
+            _paq.push(['setSiteId', 18]);
+            var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
```

Comment:
This code is hard to read because of these one letter variable names. Also I would put each statement in separate lines.